frameworks/helloworld/src/main/dist/examples/cni.yml

```diff
@@ -0,0 +1,55 @@
+name: {{FRAMEWORK_NAME}}
```

Comment:
@ArtRand was this intended to be deleted or should it be brought back as is being done here?

Reply:
I think the yaml used is overlay.yml

Reply:
Yeah that looks right, they aren't the same but `overlay.yml` appears to be a superset of `cni.yml`. I'll just remove the empty `cni.yml` instead of trying to repopulate it.